apollo-gradle-plugin/src/main/groovy/com/apollostack/android/gradle/ApolloExtension.groovy

```diff
@@ -7,29 +7,39 @@ import org.gradle.api.Project
 import org.gradle.api.tasks.util.PatternSet
 
 class ApolloExtension {
+  private static final String GRAPHQL_QUERY_PATTERN = "**/*.${GraphQLCompiler.FILE_EXTENSION}"
+  static final String NAME = "apollo"
+
   private final Project project
-  static String NAME = "apollo"
-  String graphqlPath
+  private final String sourceSet
   // Maps GraphQL query files' relative paths to a sourceSet directory
   private Multimap<String, String> cachedFiles
 
+  // Configurable extension params
```

Comment:
Those extensions are per sourceSet, can be configured as follows:

```
  sourceSets.main {
    apollo {
      graphQLPath = 'src/somethingElse'
      schemaFile = 'src/something/aSchemaFile.json'
    }
  }
```
The paths are assumed to be relative to the project directory.